bonsai/BudgetTransactions/BudgetTransactions.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  BudgetTransactions.swift
+//  bonsai
+//
+//  Created by hoang on 08.12.2021.
+//
+
+import SwiftUI
+
+struct VisualEffectView: UIViewRepresentable {
+    var effect: UIVisualEffect?
+    func makeUIView(context: UIViewRepresentableContext<Self>) -> UIVisualEffectView { UIVisualEffectView() }
+    func updateUIView(_ uiView: UIVisualEffectView, context: UIViewRepresentableContext<Self>) { uiView.effect = effect }
+}
+
+struct TransactionData: Identifiable {
+    let id = UUID()
```

Comment:
At the very end we should be using CoreData object `Transaction` on this screen
So this `TransactionData` and `TransactionItem` structures are not needed
Did you intend to use them as a mock?

Reply:
yes

Reply:
i can temporary rename to mock item

Reply:
At best I think you should use `Transaction`, for this you will need to pass DataController to Preview and create some `Transaction` instances there, then pass it to the View
An example with `Category` - https://github.com/appbonsai/bonsai./blob/7f62417e2842d8c1facad446ae0b3a7ba42e2e3e/bonsai/Modules/NewOperation/Subviews/CategoryView.swift#L36

Reply:
fixed

Reply:
👍